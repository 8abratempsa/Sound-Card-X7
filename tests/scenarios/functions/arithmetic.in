octosql "SELECT 1 + 2,
                1.0 + 2.0,
                INTERVAL 1 SECOND + INTERVAL 2 SECONDS,
                time_from_unix(1655931949) + INTERVAL 2 SECONDS,
                INTERVAL 1 SECONDS + time_from_unix(1655931949),
                'test1' + 'test2',
                1 - 2,
                -2,
                1.0 - 2.0,
                -2.0,
                INTERVAL 3 SECOND - INTERVAL 1 SECONDS,
                -INTERVAL 3 SECOND,
                time_from_unix(1655931949) - INTERVAL 2 SECONDS,
                7 * 3,
                7.0 * 3.0,
                INTERVAL 7 SECOND * 3,
                7 * INTERVAL 3 SECOND,
                'test1' * 3,
                3 * 'test1',
                /(7, 3),
                /(7.0, 3.0),
                /(INTERVAL 7 SECOND, 3),
                /(INTERVAL 7 SECOND, INTERVAL 3 SECOND)"

