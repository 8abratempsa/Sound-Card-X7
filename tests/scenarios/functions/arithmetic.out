+-------+-------+-------+------------+------------+--------------+-------+-------+-------+-------+--------+--------+------------+--------+--------+--------+--------+-------------------+-------------------+--------+--------------------+--------------+--------------------+
| col_0 | col_1 | col_2 |   col_3    |   col_4    |    col_5     | col_6 | col_7 | col_8 | col_9 | col_10 | col_11 |   col_12   | col_13 | col_14 | col_15 | col_16 |      col_17       |      col_18       | col_19 |       col_20       |    col_21    |       col_22       |
+-------+-------+-------+------------+------------+--------------+-------+-------+-------+-------+--------+--------+------------+--------+--------+--------+--------+-------------------+-------------------+--------+--------------------+--------------+--------------------+
|     3 |     3 | 3s    | 1655931951 | 1655931950 | 'test1test2' |    -1 |    -2 |    -1 |    -2 | 2s     | -3s    | 1655931947 |     21 |     21 | 21s    | 21s    | 'test1test1test1' | 'test1test1test1' |      2 | 2.3333333333333335 | 2.333333333s | 2.3333333333333335 |
+-------+-------+-------+------------+------------+--------------+-------+-------+-------+-------+--------+--------+------------+--------+--------+--------+--------+-------------------+-------------------+--------+--------------------+--------------+--------------------+
